tasks/bootstrap.yml

```diff
@@ -39,6 +39,7 @@
 
     - name: 'Restart meta0 & meta1'
       include_tasks: restart_m0m1.yml
+      when: inventory_hostname == groups[openio_directory_m0] or inventory_hostname == groups[openio_directory_m1]
```

Comment:
ca serait pas plutot un truc du genre:
`inventory_hostname in (groups[openio_directory_m0] + groups[openio_directory_m1])`

Reply:
C'est beau quand tu me parles en Ansible.